assignment/week2/웨비의사진관/index.js

```diff
@@ -0,0 +1,43 @@
+const topBtn = document.getElementById('top_btn');
+const height = document.querySelector('body');
+document.addEventListener('scroll', () => {
+  let scrollPosition = window.scrollY;
+  let scrollMax = height.scrollHeight - screen.height;
+  topBtn.style.opacity = scrollPosition / scrollMax;
+});
+
+document.addEventListener('DOMContentLoaded', () => {
+  createButton();
+});
+
+function createButton() {
+  const hover_content = document.querySelectorAll('.hover_content');
+  const hover_info = document.querySelector('.hover_info');
+
+  hover_content.forEach((item) => {
+    if (item.innerHTML.length > 75) {
```

Comment:
지금은 고정적인 `75` 값으로 구현된 것 같아요! 이렇게 되면 반응형으로 구현할 일이 생겼다고 할 때 적용이 안되겠죠..?! + 만약 75를 기준으로 구현할 일이 있다고 하더라도 이런 숫자는 상수화해주는 것이 좋답니당! ㅎㅎ
```suggestion
    if (item.clientHeight < item.scrollHeight) {
```
이런식의 방법은 어떨까요!